src/model/transaction/TransactionVersion.ts

```diff
@@ -97,4 +97,22 @@ export class TransactionVersion {
      * @type {number}
      */
     public static readonly MOSAIC_ALIAS = 1;
+
+    /**
+     * Account Property address transaction version
+     * @type {number}
+     */
+    public static readonly ACCOUNT_PROPERTY_ADDRESS_TRANSACTION = 1;
+
+    /**
+     * Account Property mosaic transaction version
+     * @type {number}
+     */
+    public static readonly ACCOUNT_PROPERTY_MOSAIC_TRANSACTION = 1;
```

Comment:
should be the same as the one introduced in TransactionType: https://github.com/nemtech/nem2-sdk-typescript-javascript/pull/73/files#diff-d4cb52883968bdb0074c79854dd7da43R103